src/components/portal/portal.tsx

```diff
@@ -128,50 +135,28 @@ export class Portal {
     private showContainer() {
         this.container.style.opacity = '1';
     }
-
     private styleContainer() {
-        if (!this.container) {
-            return;
-        }
-
         const rect: any = this.host.getBoundingClientRect();
-        const [x, y] = this.getPosition(rect);
         const viewportHeight = this.getViewportHeight();
         const containerHeight = viewportHeight - rect.y;
 
         this.container.style.position = 'absolute';
-
-        const leftX = this.openDirection === 'right' ? x : x - this.width;
-        this.container.style.left = `${leftX}px`;
-
-        this.container.style.top = `${y}px`;
-        this.container.style.width =
-            rect.width > 0 ? `${rect.width}px` : `${this.width}px`;
         this.container.style.height = `${containerHeight}px`;
         this.container.style.display = 'block';
         if (!this.visible) {
             this.container.style.display = 'none';
         }
+        this.container.style.zIndex = '7'; // Required for rendering elements above the AddDialog in the webclient
```

Comment:
We could maybe set this to some reasonable default, and provide a way to override it using a property?

Reply:
Ah, or, probably, this would make it overridable, without having to expose a property on both limel-portal and the component using it…

```suggestion
        this.container.style.zIndex = 'var(--dropdown-z-index, 7)'; // Required for rendering elements above the AddDialog in the webclient
```